CHANGELOG.md

```diff
@@ -135,6 +135,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Changed concurrent-search threadpool type to be resizable and support task resource tracking ([#7502](https://github.com/opensearch-project/OpenSearch/pull/7502))
 - Default search preference to _primary for searchable snapshot indices ([#7628](https://github.com/opensearch-project/OpenSearch/pull/7628))
 - [Segment Replication] Remove codec name string match check for checkpoints ([#7741](https://github.com/opensearch-project/OpenSearch/pull/7741))
+- Added min, max, average to resource stats in tasks API ([#7673](https://github.com/opensearch-project/OpenSearch/pull/7673))
```

Comment:
Annoying nitpick...please use the imperative mood, e.g. "Add" instead of "Added"